doc/manuals/admin/cli.md

```diff
@@ -87,6 +87,12 @@ The list of available options is the following:
     contextBroker log file, instead of re-creating it when the
     broker starts.
 -   **--silent**. Suppress all log output except errors.
+-   **--logLevel**. Select logging level, supported levels:
+    - NONE    (suppress ALL log output, including errors),
+    - ERROR   (show only error messages),
+    - WARNING (show error and warning messages),
+    - INFO    (show error, warning and informational messages),
+    - DEBUG   (show ALL messages - this is the default setting).
```

Comment:
Well, trace levels are off by default (unless you turn them on with -t).
Does Op want to remove the INFO by default?
We only have the INFO messages because Op asked us to ...
A bit confusing ...
However, I think it's a good idea to have WARNING as default, the only problem is that if somebody asks for  -t0-255  he/she will not get it, not unless he/she adds -logLevel DEBUG.
That I don't like.
Perhaps we should check traceLevels and if set, change logLevel to DEBUG?